src/core/migrations/0089_update_setting_types.py

```diff
@@ -0,0 +1,95 @@
+    ('email_subject', 'subject_notify_typesetter_proofing_changes', 'text', 'char'),
+    ('email_subject', 'subject_password_reset', 'text', 'char'),
+    ('email_subject', 'subject_peer_reviewer_pub_notification', 'text', 'char'),
+    ('email_subject', 'subject_production_assign_article', 'text', 'char'),
+    ('email_subject', 'subject_production_complete', 'text', 'char'),
+    ('email_subject', 'subject_production_manager_notification', 'text', 'char'),
+    ('email_subject', 'subject_request_revisions', 'text', 'char'),
+    ('email_subject', 'subject_reviewer_digest', 'text', 'char'),
+    ('email_subject', 'subject_review_accept_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_reviewer_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_assignment', 'text', 'char'),
+    ('email_subject', 'subject_review_complete_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_complete_reviewer_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_accept', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_decline', 'text', 'char'),
+    ('email_subject', 'subject_review_decision_undecline', 'text', 'char'),
+    ('email_subject', 'subject_review_decline_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_review_request_sent', 'text', 'char'),
+    ('email_subject', 'subject_review_withdrawl', 'text', 'char'),
+    ('email_subject', 'subject_revision_digest', 'text', 'char'),
+    ('email_subject', 'subject_section_editor_pub_notification', 'text', 'char'),
+    ('email_subject', 'subject_submission_acknowledgement', 'text', 'char'),
+    ('email_subject', 'subject_thank_proofreaders_and_typesetters', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_complete_notification', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_corrections_complete', 'text', 'char'),
+    ('email_subject', 'subject_typesetter_notification', 'text', 'char'),
+    ('email_subject', 'subject_typeset_ack', 'text', 'char'),
+    ('email_subject', 'subject_typeset_deleted', 'text', 'char'),
+    ('email_subject', 'subject_typeset_reopened', 'text', 'char'),
+    ('email_subject', 'subject_user_email_change', 'text', 'char'),
+    ('general', 'matromo_tracking_code', 'text', 'char'),
+    ('email_subject', 'subject_notify_correction_cancelled', 'text', 'char'),
+    ('general', 'copyright_submission_label', 'char', 'mini-html'),
+    ('email_subject', 'subject_accepted_review_reminder', 'text', 'char'),
+    ('email_subject', 'subject_default_review_reminder', 'text', 'char'),
+    ('email_subject', 'subject_notify_se_draft_declined', 'rich-text', 'char'),
+    ('crossref', 'crossref_date_suffix', 'text', 'char'),
+    ('general', 'default_journal_language', 'text', 'char'),
+    ('email_subject', 'subject_revisions_complete_receipt', 'text', 'char'),
+    ('email_subject', 'subject_author_copyedit_deleted', 'text', 'char'),
```

Comment:
is it worth running a single query here instead against the `'email_subject'` group?

Just so that it picks up custom emails (reminders) and those coming from plugins. We will also need to update the plugins that come with emails (I believe typesetting is the only one)

Reply:
I will put in a PR for typesetting.